fvm/src/syscalls/mod.rs

```diff
@@ -182,3 +183,9 @@ pub fn bind_syscalls(
 
     Ok(())
 }
+
+pub fn bind_validate_syscalls(
+    linker: &mut Linker<InvocationData<impl CheckedKernel + 'static>>,
+) -> anyhow::Result<()> {
+    todo!()
+}
```

Comment:
I need all the syscalls available in all contexts, even if they're no-ops. Otherwise, we can't instantiate the module.

Reply:
this will still bind all syscalls, logic of that is TODO